api/TextDecoder.json

```diff
@@ -208,50 +449,113 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextDecoder/decode",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": "38"
             },
             "chrome": {
-              "version_added": null
+              "version_added": "38"
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": "38"
             },
             "edge": {
               "version_added": null
             },
             "edge_mobile": {
               "version_added": null
             },
-            "firefox": {
-              "version_added": null
-            },
-            "firefox_android": {
-              "version_added": null
-            },
+            "firefox": [
+              {
+                "version_added": "19"
+              },
+              {
+                "version_added": "18",
+                "notes": "Slightly different version of the spec"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "19"
+              },
+              {
+                "version_added": "18",
+                "notes": "Slightly different version of the spec"
+              }
+            ],
             "ie": {
-              "version_added": null
+              "version_added": false
             },
             "opera": {
-              "version_added": null
+              "version_added": "25"
             },
             "opera_android": {
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": false
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": false
             },
             "samsunginternet_android": {
               "version_added": null
             }
           },
           "status": {
-            "experimental": false,
+            "experimental": true,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "worker_support": {
+          "__compat": {
+            "description": "Available in Web Workers",
```

Comment:
The description should be "Available in workers", see https://github.com/mdn/browser-compat-data/blob/03dbe3ccbc283469e18fa2ebdaa647e3ee752a42/schemas/compat-data-schema.md#api-specific-subfeatures